### DISCUSSION TITLE
【问答】新版本剧情怎么样？

### DISCUSSION THREAD

**Discussion by ghost** [share121/inter-knot#5422] [绝区零]
新版本剧情就看了一点还没看完问问大家这新剧情有趣吗？我怕看到后面我又开始zzz了，讨问一下好做一下心理建设
![IMG_20240926_141001](https://github.com/user-attachments/assets/a6517f75-eac6-4e5c-a78f-7f8e43ff3e89)

**Comment by Loxia1121** [comment]
又燃（物理意义）又燃（精神意义）

**Comment by fzdqz** [comment]
有大雷<img src="https://bbs-static.miyoushe.com/static/2024/05/10/0d03e0238a49d7b1e99014ea76cf45ce_6512157066528462204.png" alt="知更鸟 暗中观察" width="75" >

**Comment by 1luik** [comment]
作者的szq长度

**Comment by violet-3351** [comment]
还好传说是真的要不然真的就被刀了！！！
我的帅大叔没了！，讨厌每一个背叛者！！
最后结尾稍微说了下零号空洞是始作俑者，还有星见雅小姐💕💕
